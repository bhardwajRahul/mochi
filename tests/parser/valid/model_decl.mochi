model quick {
  provider: "openai"
  name: "gpt-3.5-turbo"
}
