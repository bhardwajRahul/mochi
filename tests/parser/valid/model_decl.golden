(program
  (model quick
    (provider (string openai))
    (name (string gpt-3.5-turbo))
  )
)
